jq-mode.el

```diff
@@ -1,4 +1,4 @@
-;;; jq-mode.el --- Edit jq scripts.
+;;; jq-mode.el --- Edit jq scripts.  -*- lexical-binding: t; -*-
```

Comment:
Does everything work with lexical-binding? I don't remember if there is anywhere the mode relies on dynamic binding. (I know I have done that in the past).

Reply:
yea, it works fine AFAICT -- plus you get the benefits/optimizations, I've never run into a problem with lexical binding in modern emacs

Reply:
Lexical binding is not the problem, it is if I have made some code that relies on dynamic binding (f.ex in the url module they do require dynamic binding of some stuff if I remember correctly), but I don't think that should be a problem here.